packages/crew_integration.rb

```diff
@@ -0,0 +1,78 @@
+require 'package'
+
+class Crew_integration < Package
+  description 'Improves integration with Chrome OS shell and Chromebrew GUI applications.'
+  homepage 'https://github.com/skycocker/chromebrew/crew_integration'
+  version '2.0'
+  source_url 'file:///dev/null'
+  source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
+  
+  depends_on "graphicsmagick"
+    
+  case ARGV[0]
+  when 'reinstall', 'install'
+     FileUtils.rm_rf "#{CREW_PREFIX}/lib/pwa/"
+     FileUtils.rm_rf "#{HOME}/MyFiles/.extension"
+  end
+    
+  case ARGV[0]
+  when 'reinstall'
+     puts "Since you are reinstalling this package, please run 'pkill ruby' if installation failed".lightblue
+  when 'upgrade'
+     puts "Since you are upgrading this package, please run 'pkill ruby' if installation failed".lightblue
+  end
+
+  def self.install
+    system "curl https://codeload.github.com/skycocker/chromebrew/zip/master -#o crew_integration.zip"
+    puts 'Archive downloaded'.lightgreen
+    system "unzip crew_integration.zip > /dev/null"
+    system "gem install em-websocket"
+    system "chmod -R +rw ./"
+    Dir.chdir 'chromebrew-master/crew_integration' do
+      system "install -Dm755 main.sh #{CREW_DEST_PREFIX}/bin/crew_integration"
+      system "install -Dm755 x-terminal-emulator.sh #{CREW_DEST_PREFIX}/bin/x-terminal-emulator"
+      system "install -Dm755 x-www-browser.sh #{CREW_DEST_PREFIX}/bin/x-www-browser"
+      FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/lib/pwa/"
+      FileUtils.ln_s "#{CREW_PREFIX}/bin/crew_integration", "#{CREW_DEST_PREFIX}/bin/pwashortcut"
+      FileUtils.mkdir_p "#{CREW_DEST_HOME}/MyFiles/.extension/"
+      FileUtils.mv Dir.glob('extension/*'), "#{CREW_DEST_HOME}/MyFiles/.extension/"
+      FileUtils.mv Dir.glob('*'), "#{CREW_DEST_PREFIX}/lib/pwa/"
+      FileUtils.ln_s "#{CREW_PREFIX}/lib/pwa/send.rb", "#{CREW_DEST_PREFIX}/lib/pwa/sender.rb"
+    end 
+  end
+  def self.postinstall
+      case ARGV[0]
+      when 'reinstall', 'install'
```

Comment:
Why is this necessary?  If you run `crew postinstall crew_integration`, you won't see this message?

Reply:
We don't need this part at upgrade as it doesn't need to re-complete the setup again, added `postinstall` now